File: Liguagem C#/1010.cs
using System; 

class URI {

    static void Main(string[] args) {
            string[] input;
            input = Console.ReadLine().Split(' ');
        
        int codigoPeca1 = int.Parse(input[0]);
        int numeroPeca1 = int.Parse(input[1]);
        double valorPeca1 = double.Parse(input[2]);
        
            input = Console.ReadLine().Split(' ');
        
        int codigoPeca2 = int.Parse(input[0]);
        int numeroPeca2 = int.Parse(input[1]);
        double valorPeca2 = double.Parse(input[2]);
        
        double totalPagar = (numeroPeca1 * valorPeca1) + (numeroPeca2 * valorPeca2); 
    
        Console.Write("VALOR A PAGAR: R$ {0:F2}\n", totalPagar);
    }
}

File: Linguagem Dart/1010.dart
import 'dart:io';

void main()
{   

    List<String> linha1 = stdin.readLineSync().split(" ");
    
    int codigoPeca1 = int.parse(linha1[0]);
    int numeroPeca1 = int.parse(linha1[1]);
    double valorPeca1 = double.parse(linha1[2]);
    
    List<String> linha2 = stdin.readLineSync().split(" ");
    
    int codigoPeca2 = int.parse(linha2[0]);
    int numeroPeca2 = int.parse(linha2[1]);
    double valorPeca2 = double.parse(linha2[2]);
    
    double valorTotal = (numeroPeca1 * valorPeca1) + (numeroPeca2 * valorPeca2);
    
    print("VALOR A PAGAR: R\$ ${valorTotal.toStringAsFixed(2)}");
    
}
